bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/votable/VoTableParser.scala

```diff
@@ -140,24 +178,25 @@ trait VoTableParser {
       k.sequenceU
     }
 
-    def combineWithErrors(m: List[Magnitude], e: Map[MagnitudeBand, Double]) = m.map(i => i.copy(error = e.get(i.band)))
+    def combineWithErrorsAndFilter(m: List[Magnitude], e: Map[MagnitudeBand, Double]): List[Magnitude] =
+      m.map(i => i.copy(error = e.get(i.band))).filterNot(magnitudesFilter.nonValidMagnitude)
```

Comment:
I'd think it is faster `nonValidMagintude` as it only needs to check 1 invalid options while a `validMagnitude` would need to check a bunch. Basically `all - 1` (amag)